core/functions/datetime.rs

```diff
@@ -362,6 +350,16 @@ fn to_julian_day_exact(dt: &NaiveDateTime) -> f64 {
     jd_days + jd_fraction
 }
 
+fn format_julian_day(days: f64) -> String {
+    let t = (days * 100_000_000.0).round() / 100_000_000.0;
+    let mut ret = format!("{}", t);
+    if !ret.contains('.') {
+        ret += ".0";
+    }
+
+    ret
+}
```

Comment:
please comment this function and extract `100_000_000.0` to a constant with a name to indicate that it is used for 8-decimal precision